plugin/src/main/java/com/denizenscript/denizen/events/entity/EntityBreedScriptEvent.java

```diff
@@ -40,6 +39,12 @@ public class EntityBreedScriptEvent extends BukkitScriptEvent implements Listene
 
     public EntityBreedScriptEvent() {
         registerCouldMatcher("<entity> breeds");
+        this.<EntityBreedScriptEvent, ElementTag>registerDetermination(null, ElementTag.class, (evt, context, determination) -> {
+            if (determination.isInt()) {
+                experience = determination.asInt();
+                evt.event.setExperience(experience);
+            }
+        });
```

Comment:
This should be `registerOptionalDetermination`.
Also there's probably no need for the `experience` variable vs. just passing `determination.asInt()` directly?